src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -73,58 +58,50 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   // Send a user to datastore. Update the current information about the user if one exists.
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGitHubID = request.getParameter("githubID");
-    if (userGitHubID == null || userGitHubID.trim().length() == 0) {
+    String userGitHubAuthToken = request.getParameter("gitHubAuthToken");
```

Comment:
nit: this can just be called `gitHubAuthToken`. the `user` part of the name is implied, because there is almost no scenario where a servlet should (or even could) have the auth token for a github account that doesn't belong to the current user.